test/runtests.jl

```diff
@@ -1,7 +1,12 @@
 using Test
 using RSEinJulia
+using Aqua: Aqua
 
 @test_nowarn greet()
 
+@testset "Aqua.jl" begin
+    Aqua.test_all(RSEinJulia)
```

Comment:
```suggestion
    # We do not test `project_extras` since we do not follow the established
    # practice to have the same name of the repository as the Julia package
    # defined therein.
    Aqua.test_all(RSEinJulia; project_extras = false)
```